examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -17,8 +17,8 @@
 print('Functional nifti image (4D) is located at: %s' %
       haxby_dataset.func)
```

Comment:
haxby.func is a list of 4d nifti filenames, It doesn't seem like a good idea to use that directly and I am actually surprised you don't get an error somewhere...